src/release/installer/win/GeoServerEXE.nsi

```diff
@@ -608,7 +608,7 @@ Function Port
   Pop $PortHWND
   ${NSD_OnChange} $PortHWND PortCheck
 
-  ${NSD_CreateLabel} 110u 40u 120u 14u "Valid range is 1024-65535." 
+  ${NSD_CreateLabel} 110u 40u 120u 14u "Valid range is 80, 1024-65535." 

```

Comment:
Because all ports are not valid, ports below 1024 are protocol specific - 80 happens to the one for http.